tests/ffi/memory/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eux
+
+# shellcheck disable=SC1091
+
+. ../memory/prepare.sh
+
+prepare_test
+reload_config
+
+podman run --rm --replace -d --name ffi-asil \
+     quay.io/centos-sig-automotive/ffi-tools:latest \
+     ./ASIL/20_percent_memory_eat > /dev/null
+# Copy container image registry to /var/qm/lib/containers
+image_id=$(podman images | grep quay.io/centos-sig-automotive/ffi-tools | awk -F " " '{print $3}')
+mkdir -p /var/qm/lib/containers/registry
```

Comment:
I would recommend have a global var session and check if dir already exist before creating it.

QM_REGISTRY_DIR="/var/qm/lib/containers/registry"
if [ ! -d "${QM_REGISTRY_DIR}" ]; then
    mkdir -p ${QM_REGISTRY_DIR}"
fi

Reply:
Ack